tests/test_fork_provider.py

```diff
@@ -71,6 +61,20 @@ def create_fork_provider(network_api, port):
     return provider
 
 
+@alchemy_xfail
+def test_request_timeout(mainnet_fork_provider, config, network_api):
```

Comment:
You were not accepting the necessary fixtures before which caused things not to work.